js/app/widgets/eknWebview.js

```diff
@@ -176,9 +177,31 @@ const EknWebview = new Knowledge.Class({
                 if (scheme === 'license') {
                     let license = GLib.uri_unescape_string(uri.replace('license://', ''), null);
                     uri = Endless.get_license_file(license).get_uri();
+
+                    let license_view = new EosKnowledgePrivate.WebViewDialog({
+                      'transient-for' : this.get_toplevel(),
+                      'width-request' : 640,
+                      'height-request': 720,
```

Comment:
Not sure these make a lot of sense, since we want to be compatible with composite screens. You can set the default width and height though.